test/integration/framework/util.go

```diff
@@ -45,20 +45,20 @@ const (
 )
 
 // CreateNamespaceOrDie creates a namespace.
-func CreateNamespaceOrDie(c clientset.Interface, baseName string, t *testing.T) *v1.Namespace {
+func CreateNamespaceOrDie(c clientset.Interface, baseName string, tb testing.TB) *v1.Namespace {
 	ns := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: baseName}}
 	result, err := c.CoreV1().Namespaces().Create(context.TODO(), ns, metav1.CreateOptions{})
 	if err != nil {
-		t.Fatalf("Failed to create namespace: %v", err)
+		tb.Fatalf("Failed to create namespace: %v", err)
```

Comment:
Nit (up to you): changing the name of the variable increases the diff unnecessarily. Most places in the codebase use `t` as a variable name for `testing.TB`, but there is no consistency.

Reply:
Yeah, it looks like other usages of TB also just use the variable `t`